CardScan/Classes/ScanBaseViewController.swift

```diff
@@ -184,44 +191,37 @@ public protocol TestingImageDataSource: AnyObject {
         self.videoFeed.requestCameraAccess(permissionDelegate: self)
     }
     
-    func setVideoOrientation() {
-        if ScanBaseViewController.isPadAndFormsheet {
-            self.previewView?.videoOrientation = AVCaptureVideoOrientation(rawValue: UIWindow.interfaceOrientation.rawValue)
-            self.videoFeed.videoOrientation = self.previewView?.videoOrientation
-        } else {
-            UIDevice.current.setValue(UIDeviceOrientation.portrait.rawValue, forKey: "orientation")
-            self.previewView?.videoOrientation = .portrait
-            self.videoFeed.videoOrientation = .portrait
-        }
-    }
-    
     public func setupOnViewDidLoad(regionOfInterestLabel: UIView, blurView: BlurView, previewView: PreviewView, cornerView: CornerView?, debugImageView: UIImageView?, torchLevel: Float?) {
         
         self.regionOfInterestLabel = regionOfInterestLabel
         self.blurView = blurView
         self.previewView = previewView
         self.debugImageView = debugImageView
         self.cornerView = cornerView
+        ScanBaseViewController.isPadAndFormsheet = UIDevice.current.userInterfaceIdiom == .pad && self.modalPresentationStyle == .formSheet
         
         setNeedsStatusBarAppearanceUpdate()
         regionOfInterestLabel.layer.masksToBounds = true
         regionOfInterestLabel.layer.cornerRadius = self.regionOfInterestCornerRadius
         regionOfInterestLabel.layer.borderColor = UIColor.white.cgColor
         regionOfInterestLabel.layer.borderWidth = 2.0
-  
-        self.ocrMainLoop()?.mainLoopDelegate = self
-        self.previewView?.videoPreviewLayer.session = self.videoFeed.session
-        ScanBaseViewController.isPadAndFormsheet = UIDevice.current.userInterfaceIdiom == .pad && self.modalPresentationStyle == .formSheet
-        
-        self.setVideoOrientation()
+    
+        if !ScanBaseViewController.isPadAndFormsheet {
+            UIDevice.current.setValue(UIDeviceOrientation.portrait.rawValue, forKey: "orientation")
+        }
         
         if testingImageDataSource != nil {
             self.ocrMainLoop()?.imageQueueSize = 20
         }
         
+        self.ocrMainLoop()?.mainLoopDelegate = self
+        self.previewView?.videoPreviewLayer.session = self.videoFeed.session
+        
+        self.videoFeed.videoOrientation = self.initialVideoOrientation
```

Comment:
I suggest getting rid of the `videoOrientation` variable in `videoFeed` and just pass the initial orientation into the setup function -- I think that it only gets used there.